cypress/integration/blockquote.spec.js

```diff
@@ -33,4 +33,226 @@ describe( 'Blockquote Block', () => {
 		'Seren Blockquote',
 		'Yule Blockquote',
 	] ) )
+	it( 'should adjust desktop options in style tab', () => {
+		cy.setupWP()
+		cy.newPage()
+		cy.addBlock( 'ugb/blockquote' )
+		cy.openInspector( 'ugb/blockquote', 'Style' )
+
+		//Test General Alignment
+		cy.collapse( 'General' )
+		const aligns = [ 'left', 'center', 'right' ]
+		aligns.forEach( align => {
+			cy.adjust( 'Align', align ).assertComputedStyle( {
+				'.ugb-inner-block': {
```

Comment:
i see that you used single quote for element selector and backticks for CSS rules. this is better. will do a PR to change my columns tests